### ISSUE TITLE
Integration

### ISSUE THREAD

**Issue by ThexXTURBOXx** [ThexXTURBOXx/Reforged-Integration#1]
Here is a little list for mods, which will get integrated then:
- [x] Thaumcraft
- [x] ProjectE
- [x] Botania 1.8.9 Unofficial
- [x] Biomes O' Plenty
- [ ] Buildcraft
- [ ] Copper, Tin etc. Mod-independent
- [ ] Psi (Maybe new "Spells", too. Something like: Shoot Boomerang with a vector as input)
- [ ] Aura Cascade
- [ ] JEI

And last but not least, here is a little list of other things to do:
- [ ] A class for other modders to easily register their own addons in our mod (a class with methods like createBoomerang(ToolMaterial, ...) [Maybe Later]

**Comment by Sunconure11** [comment]
I'll create new textures for thaumcraft void metal weaponry, and create thaumium ones later on.

**Comment by Sunconure11** [comment]
I need to deal with Growthcraft, and I have stretched myself out fairly far. I have to have someone else finish up my work for Baykok for Totemic at the moment.

**Comment by ThexXTURBOXx** [comment]
I will finish the Crossbow in the Reforged mod and then I will try to integrate some more mods (0.3-alpha is coming)

**Comment by Sunconure11** [comment]
After I finish the thaumium textures, and after I texture a specific model, I'll texture manasteel textures. Not sure if I should do terrasteel and elementium.

**Comment by ThexXTURBOXx** [comment]
Ok, I would like to include terrasteel and elemntium, too. The only problem is: what special abilites should they have? I will think a little bit again and then I will say you if I include them, too.

**Comment by Sunconure11** [comment]
FYI, NEI is no longer dominant. Remove NEI from this list.

I think I have old copper textures.

**Comment by ThexXTURBOXx** [comment]
Ok, I did

**Comment by Sunconure11** [comment]
Vazkii has taken over Botania once more, therefore, integration with Botania would have to be handled a bit differently. Willie's fork could be used for 1.8 and 1.9, but Vazkii's version would have to be for 1.10.2.

**Comment by ThexXTURBOXx** [comment]
Yeah, I have seen it and played a little bit with it. I will probably integrate it after the 0.7-compatibility update

**Comment by Sunconure11** [comment]
List of mods that are no longer on 1.12, in the event this is picked up again:

Thaumcraft
Aura Cascade

List of materials that mods have since removed:

BoP has removed Amethyst as a tool material

**Comment by ThexXTURBOXx** [comment]
I've noticed BoP's change with the Amethyst being removed, I already wrote code to bypass this (it got removed at any 1.9 version, I think. So it will get integrated, if it's possible).
I will just remove the other mods' integrations.

**Comment by ghost** [comment]
Tinker's Construct!
